xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexAdapters.java

```diff
@@ -706,7 +696,16 @@ public static ExchangeMetaData adaptMetaData(
   }
 
   public static ExchangeMetaData adaptMetaData(
-      BitfinexAccountFeesResponse accountFeesResponse, ExchangeMetaData metaData) {
+          BitfinexAccountFeesResponse accountFeesResponse, Integer platformStatus, ExchangeMetaData metaData) {
+    final WalletHealth health;
+    if (platformStatus == PLATFORM_STATUS_ONLINE) {
```

Comment:
can you change this from Integer to int, this can NPE

Reply:
Okay

Reply:
still an integer?

Reply:
This is not how this will work if you compare a null to an Int you will just blow up here, you need a better way of handling this.